app/services/host_aspects.rb

```diff
@@ -0,0 +1,11 @@
+module HostAspects
```

Comment:
This class represents configuration for all available aspects, so plural form is kind of logical here.